### PR TITLE
Feature/documentation sighandler keycert

### PR DESCRIPTION
**Pull request by Razumain**
Documentation and minor field naming errors for key and cert module implementations and for sig handler

### CODE REVIEW

File: docs/what-is.md [hunk 1]
```diff
@@ -2,7 +2,122 @@
...
+
+## Overview
+
+A signature service is the central service component in a remote signing setup (also known as Federated Signing) as illustrated
+in this figure. This setup is characterized by a signature process that involves a number of independent services and actors with
+distinct roles:
+
+| Role                    | Description                                                                                                                            |
+|:------------------------|:---------------------------------------------------------------------------------------------------------------------------------------|
+| Signer                  | A natural person that is signing a document.                                                                                           |
```
Comment (martin-lindstrom) [inline docs/what-is.md:15]: We are not writing specs. I suggest we remove "natural".

File: docs/what-is.md [hunk 2]
```diff
@@ -2,7 +2,122 @@
...
+| Service Provider        | The service visited by the signer where the signer reviews data to be signed and agrees to sign.                                       |
```
Comment (martin-lindstrom) [inline docs/what-is.md:16]: This is also referred to as a "Signature Requestor"....

File: docs/what-is.md [hunk 3]
```diff
@@ -2,7 +2,122 @@
...
+
+Traditional remote signature services store a static persistent signing key for each natural person signer. Each such key is
+associated with a static and persistent certificate attached to all signatures created by a particular signer. This 
+setup requires that the signing service has a persistent relationship with each signer and has means to ensure
+that each signing key is never ever used by more than one signer.
+
+The federated signing model (implemented by Sweden Connect Signing Service) avoids this significant management overhead by always generating a new key and a new
+signing certificate for each instance of signing and by immediately and permanently destroying all knowledge
+about the signing key after completed signing. In this setup there is no need to have persistent user-accounts for storing
+private signing keys and to prevent multiple signer's from using the same key. This is replaced by a process
```
Comment (martin-lindstrom) [inline docs/what-is.md:32]: "multiple signer's" -> "multiple signers".

**Review by martin-lindstrom — CHANGES_REQUESTED** [review]
Some changes needed. I gave up the detailed review. Will go through everything before release.

File: docs/what-is.md [hunk 4]
```diff
@@ -2,7 +2,122 @@
 
 # What is a Signature Service?
 
-> TODO: An introduction to what a signature service is. A few illustrative figures may also be helpful.
+![SignServiceOverview](images/signservice-overview.svg)
```
[the remark this answers is not in the document]
Comment (Razumain) [reply docs/what-is.md]: Fixed

**Review by martin-lindstrom — APPROVED** [review]

**Pull request merged**